go.mod

```diff
@@ -170,6 +170,7 @@ require (
 	google.golang.org/grpc v1.47.0 // indirect
 	google.golang.org/protobuf v1.28.0 // indirect
 	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
compose-go needs to be fixed too
- https://github.com/compose-spec/compose-go/pull/153

Reply:
@AkihiroSuda  https://github.com/compose-spec/compose-go/pull/153 is still Draft, can we merge this before compose-go ?

Reply:
Yes